fdbserver/DataDistributionTracker.actor.cpp

```diff
@@ -300,8 +300,12 @@ ACTOR Future<Void> readHotDetector(DataDistributionTracker* self) {
 			loop {
 				try {
 					Standalone<VectorRef<KeyRangeRef>> readHotRanges = wait(tr.getReadHotRanges(keys));
+					TraceEvent("ReadHotRangeLog").detail("Ranges", readHotRanges.size());
```

Comment:
@dongxinEric was also doing something related to this. He might want to weigh in here.